processor/probabilisticsamplerprocessor/murmur3_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package probabilisticsamplerprocessor
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal/idutils"
+)
+
+// Test_hash ensures that the hash function supports different key lengths even if in
+// practice it is only expected to receive keys with length 16 (trace id length in OC proto).
+func Test_hash(t *testing.T) {
```

Comment:
Is this just ensuring there are no collisions? If so, the test name could be "TestHashHasNoCollisions" (which is impossible to prove).

Reply:
This is just a refactor from the tests that were there before as we moved the hashing algo to its own file. I can certainly rename it.